src/modules/MouseUtils/FindMyMouse/FindMyMouse.cpp

```diff
@@ -54,6 +54,9 @@ struct SuperSonar
     HWND m_hwnd;
     POINT m_sonarPos = ptNowhere;
 
+    // Only consider double left control click if at least 100ms passed between the clicks, to avoid keyboards that might be sending rapid clicks.
+    static const int MIN_DOUBLE_CLICK_TIME = 100;
```

Comment:
Should be `GetDoubleClickTime() / 5` (recalculated on the fly, not a constant). Otherwise, if somebody sets their double-click time to 100ms, they will never be able to activate.

Reply:
Is it even possible to do it in less that 100 ms?
No harm in adding that change, though, just in case.
I'll do it. Thank you.

Reply:
@oldnewthing I've added the change here: https://github.com/microsoft/PowerToys/pull/14291